projects/angular-gridster2/src/lib/gridster.component.ts

```diff
@@ -60,6 +60,11 @@ export class GridsterComponent
   previewStyle$: EventEmitter<GridsterItem | null> =
     new EventEmitter<GridsterItem | null>();
 
+  calculateLayout$ = new Subject<void>();
+
+  private resize$ = new Subject<void>();
+  private destroy$ = new Subject<void>();
```

Comment:
I think you forgot to call complete on destroy$ in ngOnDestroy

Reply:
Thanks!